app/src/main/java/org/example/ChessPieces/ChessPiece.java

```diff
@@ -0,0 +1,28 @@
+package org.example.ChessPieces;
+
+import org.example.BoardPosition;
+
+// A generic class for all chess pieces.
+public abstract class ChessPiece {
+    private char mDisplayCharacter;
+    private boolean mIsWhite;
+
+    public ChessPiece(char pieceDisplayCharacter) {
+        mDisplayCharacter = pieceDisplayCharacter;
+        mIsWhite = mDisplayCharacterIsWhite(pieceDisplayCharacter);
+    }
+
+    private boolean mDisplayCharacterIsWhite(char pieceDisplayCharacter) {
```

Comment:
This method doesn't reference anything from the class, which means it can and should be a static method.

Reply:
Moved this to the constructor altogether since it's a private method and is only called from the constructor anyway.